tools/ci/inria/other-configs/script

```diff
@@ -36,9 +36,7 @@ ${main} -conf --disable-native-compiler \
         -conf --disable-unix-lib \
         -conf --disable-bigarray-lib \
         -conf --disable-ocamldoc \
-        -conf --disable-native-compiler \
-        -conf --disable-dependency-generation \
-        -no-native
```

Comment:
I didn't check exactly what difference it makes, but I don't think you mean to delete the `-no-native` line as well?

Reply:
Oops - sorry for noise, I see from the commit message that you _did_ mean to remove it!!